src/main/java/de/teamlapen/vampirism/entity/converted/BiteableEntryManager.java

```diff
@@ -94,6 +97,57 @@ BiteableEntry calculate(@Nonnull CreatureEntity creature) {
         }
     }
 
+    /**
+     * checks if the entity type is blacklisted for blood calculation
+     * does not check if the entity is blacklisted in the server config ({@link #isEntityTypeBlacklisted(EntityType)})
+     *
+     * @param type the entity type
+     * @returns weather the entity type is blacklisted or not
+     */
+    public boolean isEntityTypeBlacklistedByDefault(EntityType<?> type) {
+        if (!VampirismConfig.SERVER.autoCalculateEntityBlood.get()) return true;
+        if (!(type.getClassification() != EntityClassification.MONSTER && type.getClassification() != EntityClassification.WATER_CREATURE))
+            return true;
+        if (ModTags.Entities.VAMPIRE.contains(type)) return true;
+        return false;
+    }
+
+    /**
+     * checks if the entity type is blacklisted for blood calculation
+     *
+     * @param type the entity type
+     * @returns weather the entity type is blacklisted or not
+     */
+    public boolean isEntityTypeBlacklisted(EntityType<?> type) {
```

Comment:
Not really sure if we really need four small methods here. Since the blacklist is specifically for blood calculations the methods won't likely be used elsewhere, so no need to differentiate. One or two just make things much more clear.

Reply:
squashed